src/main/resources/job-scheduler-config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "projectId": "client-side-events",
+  "configuration":
+  {
+    "query":
+    {
+      "writeDisposition": "WRITE_TRUNCATE",
+      "destinationTable":
+      {
+        "projectId": "client-side-events",
+        "datasetId": "Aggregate_Tables"
+      }
+    }
+  },
+
+  "every4Hours": [
```

Comment:
I think it would be better to have an "hours" field for each query instead of separating them into two groups of 4 and 24.  We don't need an "interval" parameter on the job.  We can run the job every hour, and then check the table config to see which tables should run at that hour of the day.  
Up to you if you want to do that at this point.